src/main/java/model/figure/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package model.figure;
+
+import model.Figure;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: Circle, area:  " + calculateArea()
```

Comment:
The message printed in the `print` method incorrectly states the figure as a 'Circle' instead of 'RightTriangle'.